test/bootloader_test.rb

```diff
@@ -0,0 +1,146 @@
+require_relative "test_helper"
+
+Yast.import "Bootloader"
+
+describe Yast::Bootloader do
+  subject { Yast::Bootloader }
+
+  describe ".Import" do
+    # FIXME: looks useless as import is used in autoinstallation and in such case reset do nothing
+    it "resets configuration" do
+      expect(subject).to receive(:Reset)
+
+      subject.Import({})
+    end
+
+    it "marks that configuration is read" do
+      subject.Import({})
+
+      expect(Yast::BootCommon.was_read).to eq true
+    end
+
+    it "marks that configuration is already proposed" do
+      subject.Import({})
+
+      expect(Yast::BootCommon.was_proposed).to eq true
```

Comment:
Even though, from a unit testing POV, if we are not enforcing/checking the previous value we cannot actually assert that "it marks that configuration is read".